content/schemas/sign_request_create_signer.yml

```diff
@@ -0,0 +1,112 @@
+---
+title: Signer fields for Create Sign Request
+
+type: object
+
+description: |-
+  The schema for a Signer in a POST Sign Request request body
+
+required:
+  - email
+
+properties:
+  email:
+    type: string
+    description: Email address for signer
+    example: example@gmail.com
+
+  first_name:
+    type: string
+    description: First name of signer
+    example: Aaron
+
+  last_name:
+    type: string
+    description: Last name of signer
+    example: Levie
+
+  role:
+    type: string
+    enum:
+      - signer
+      - approver
+      - subscriber
+      - final_copy_reader
+    description: Defines the role of the signer in the sign request. Signers will
+      need to sign the document, Approvers may just approve the document,
+      and subscribers will not need to take any actions but will be notified
+      of each action. Finally, final_copy_reader will only receive the finished
+      sign request with a sign log
+    example: signer
+
+  is_in_person:
+    type: boolean
+    description: 'When used in combination with an embed url on the sender,
+      after sender has signed, they will be redirected to the next in_person
+      signer, see: In person signing'
+    example: true
+
+  order:
+    type: integer
+    description: Order of signer
+    minimum: 0
+    example: 2
+
+  language:
+    type: string
+    enum:
+      - en
```

Comment:
I don't know if we want to try to standardize on Box's language codes: https://developer.box.com/guides/api-calls/language-codes/.  I'm fine either way.. just wanted it to be considered.

Reply:
Try to standardize

Reply:
We can only support the languages that Sign Request supports here https://signrequest.com/api/v1/docs/#section/Working-with-a-SignRequest/Signer-language but I can update the languages that overlap to use the same enum value? Or by standardize did you mean remove any languages that aren't supported by box?

Reply:
The former - shouldn't take away languages we only support in sign request but not in box.